packages/lesmis/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@tupaia/lesmis",
+  "version": "0.1.0",
+  "private": true,
+  "description": "Lao PDR Education and Sports Management Information System",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/beyondessential/tupaia",
+    "directory": "packages/lesmis"
+  },
+  "author": "Beyond Essential Systems <admin@tupaia.org> (https://beyondessential.com.au)",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "react-scripts build",
+    "postbuild": "rm -rf served_build && mv build served_build",
+    "eject": "react-scripts eject",
+    "lint": "eslint --ignore-path ../../.gitignore .",
+    "lint:fix": "yarn lint --fix",
+    "start-dev": "react-scripts start",
+    "start-fullstack": "npm-run-all -c -l -p start-meditrak-server start-web-config-server start-dev",
```

Comment:
Will `lesmis` be relying on `web-config-server`? Or is this just placeholder for now?

Reply:
Thanks Rohan. It's just a placeholder for now. There will be a `lesmis-server` but Edwin suggests that it will be relatively similar to `web-config-server` especially in terms of the `auth` and the api for the `dashboards`